source/localizable/tutorial/location-service.md

```diff
@@ -0,0 +1,301 @@
+In Super Rentals, we want the ability to provide directions from where the renter is located to his or her various rental options.
+To accomplish this, we need to make use of the browser's native geolocation API to determine our current location,
+as well as interface with google's map and geocoding API to display directions.
+
+To implement this feature, we are going to take advantage of several Ember concepts:
+
+1. A component to display a google map with directions.
+2. Utility functions to interface with Google APIs.
+3. A service to keep track of the browser's current location.
+
+### Making Google Maps Available in Ember
+
+There are several ways to include 3rd party libraries in Ember. See the guides section on [managing dependencies](../../addons-and-dependencies/managing-dependencies/) as a starting point when you need to add one.
+
+Since Google provides its map API as a remote script, we'll use curl to download it into our project's vendor directory.
+
+```shell
+curl -o vendor/gmaps.js https://maps.googleapis.com/maps/api/js?v=3.22
+```
```

Comment:
The file will end up in the wrong spot if they aren't in the right folder in their Terminal, correct?  For Windows users (who don't have `curl`) this may be confusing.  I wonder if we should just tell them to download the contents and put them where we tell them to instead?